example/lstm.py

```diff
@@ -506,18 +505,15 @@ def train_lstm(
     # Dict name (string) -> numpy ndarray
     params = init_params(model_options)
 
-    if init:
-        if reload_model:
-            load_params('lstm_model.npz', params)
+    if reload_model:
+        load_params('lstm_model.npz', params)
 
     # This creates Theano Shared Variable from the parameters.
     # Dict name (string) -> Theano Tensor Shared Variable
     # params and tparams have different copy of the weights.
     tparams = init_tparams(params)
 
-    worker.init_shared_params('DLTlstm', tparams.values(),
-                              param_sync_rule=EASGD(0.5),
-                              cleanup=init)
+    worker.init_shared_params(tparams.values(), param_sync_rule=ASGD())
```

Comment:
Because we aren't able to get any training speed up with EASGD now. So we should more recommand to use ASGD